drivers/sd/bcm2836/rpisdhc/rpisdhc.cpp

```diff
@@ -1348,12 +1361,37 @@ NTSTATUS SDHC::sendCommandInternal (
 
         status = this->getLastCommandCompletionStatus();
         if (!NT_SUCCESS(status)) {
+            //
+            // Need to read status register again to get real error code
+            //
+            _HSTS hsts{ 0 };
+            this->readRegisterNoFence(&hsts);
+            //
+            // CMD1 alwary return CRC7 error on EMMC device
+            // Clear all ERROR mask set and return Success
+            // Temp clear error flags for read/write test command
+            //
+            if(((Cmd.Fields.Command == 0x1) && (hsts.Fields.Crc7Error))) //||
```

Comment:
Can we remove the commented out code? Or is there a reason you are keeping it here?